openstf/model/basecase.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from datetime import datetime, timedelta, timezone
+import pytz
+import numpy as np
+import pandas as pd
+
+
+from sklearn.base import BaseEstimator, RegressorMixin
+
+MINIMAL_RESOLUTION: int = 15  # Used for validating the forecast input
+
+
+class BaseCaseModel(BaseEstimator, RegressorMixin):
+    def predict(self, forecsst_input_data: pd.DataFrame) -> pd.DataFrame:
+        """Predict using the basecase method
```

Comment:
Being a little picky; 'Predict using the basecase method' can be more clear -> Something to indicate that this class matches the sklearn class